src/realmCryo.ttl

```diff
@@ -261,7 +493,15 @@ soreac:Tundra rdf:type owl:Class ;
                               owl:onProperty sorel:hasRealm ;
                               owl:someValuesFrom soreac:FrozenGround
                             ] ;
-            rdfs:label "tundra"@en .
+            rdfs:label "tundra"@en ;
+            skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_00000112> ;
+            skos:definition  [
+              rdfs:comment  "A vegetated area which is part of a tundra ecosystem."@en ;
```

Comment:
if this is the definition, shouldn't there be an axiom like 'partOf only "tundra ecosystem"'  I can't find 'tundra ecosystem' in SWEET. What does it mean?  
Maybe Wikipedia " tundra is a type of biome where the tree growth is hindered by low temperatures and short growing seasons. "

Reply:
another SWEET refinement problem.